src/main/java/com/microsoft/sqlserver/jdbc/SQLServerStatement.java

```diff
@@ -977,6 +980,31 @@ final void resetForReexecute() throws SQLServerException {
             return false;
         return temp.substring(0, 6).equalsIgnoreCase("select");
     }
+    
+    /**
+     * Determine if the SQL is a INSERT.
+     * 
+     * @param sql
+     *            The statment SQL.
+     * @return True is the statement is a select.
```

Comment:
"True is the statement is a select."?

Reply:
Done. Also changed the javadocs for isSelect (which my javadoc came from).